packages/amplify-cli/src/cli.js

```diff
@@ -19,10 +20,29 @@ async function run(argv) {
   // and run it
   const context = await cli.run(argv);
 
+  await askAmplifyMetricsQuestion(context);
+
   // send it back (for testing, mostly)
   return context;
 }
 
+async function askAmplifyMetricsQuestion(context) {
+  const projectConfigFilePath = context.amplify.pathManager.getProjectConfigFilePath();
+  const projectConfig = JSON.parse(fs.readFileSync(projectConfigFilePath));
+
+  if (projectConfig.sendAmplifyMetrics === undefined) {
+    if (await context.prompt.confirm('Do you want to send metrics to AWS Amplify?')) {
```

Comment:
We need to update this text to clarify anonymous:

"We will occasionally send anonymous usage statistics to the AWS Amplify team to make the tool better."

Reply:
"We will occasionally send anonymous usage statistics to the AWS Amplify team to make the tool better. Do you want to send metrics to the AWS Amplify team?"

How about this?

Reply:
Updated the text.